appsync-data-with-images/README.md

```diff
@@ -0,0 +1,60 @@
+# AWS Service 1 to AWS Service 2
+
+This pattern << explain usage >>
+
```

Comment:
Looks like this README isn't updated for this pattern with deployment instructions